ga4gh/cli.py

```diff
@@ -521,7 +552,7 @@ def addNameArgument(parser):
 def addClientGlobalOptions(parser):
     parser.add_argument('--verbose', '-v', action='count', default=0)
     parser.add_argument(
-        "--workarounds", "-w", default='', help="The workarounds to use")
```

Comment:
No, the old way would create a set of workarounds initialized to the set containing an empty string.  I think this is a better pattern to follow generally, since when sending arguments over the wire, sending an empty string instead of a null is usually not what you want.  (Although I agree it matters less in the specific workarounds case.)